rubin_sim/maf/batches/scienceRadarBatch.py

```diff
@@ -503,6 +503,30 @@ def scienceRadarBatch(
         temp_list.append(bundles[b])
     bundleList.extend(temp_list)
 
+    # XRB metric
+    displayDict["subgroup"] = "XRB"
+    n_events = 10000
+    slicer = generateXRBPopSlicer(n_events=n_events)
+    metric = XRBPopMetric(outputLc=True)
+    xrb_summaryMetrics = [
```

Comment:
Isn't this set just the "lightcurveSummary"? https://github.com/lsst/rubin_sim/blob/main/rubin_sim/maf/batches/common.py#L151

Reply:
That said, I think I'd add one more metric on top of these that is just the "MedianMetric" and just call it the median .. that would likely be useful for the reduced "Number of detections" and "Time between detections" and it would also have a straight forward name (it'd be useless for the ever detect / early detect, but for the other two would be ok and clearer)